pkgs/development/tools/rufo/default.nix

```diff
@@ -0,0 +1,16 @@
+{ bundlerApp, bundlerUpdateScript, lib }:
+
+bundlerApp {
+  pname = "rufo";
+  gemdir = ./.;
+  exes = [ "rufo" ];
+
+  passthru.updateScript = bundlerUpdateScript "rufo";
```

Comment:
Do we still need `bundlerUpdateScript` here?  My understanding is that `bundlerApp` will already arrange for the needed gemsets here and expose only the `exes` above, so there isn't a need to call bundix further.

Reply:
This has nothing to do with arranging for the needed gemsets; it’s for `./maintainers/scripts/update.nix`, which empirically does work with this line and does not work without it.

Reply:
Thanks for clarifying!